nautobot_golden_config/templates/nautobot_golden_config/compliance_device_report.html

```diff
@@ -122,7 +122,7 @@ <h1>{% block title %}Configuration Compliance - {{ device.name }}{% endblock %}<
             <tr>
                 <td style="width:250px">Actual Configuration</td>
                 <td class="config_hover">
-                    <span id="{{ item.rule|slugify }}_intended"><pre>{{ item.intended|placeholder }}</pre></span>
+                    <span id="{{ item.rule|slugify }}_intended"><pre>{{ item.actual|placeholder }}</pre></span>
                     <span class="config_hover_button">
                         <button class="btn btn-inline btn-default hover_copy_button" data-clipboard-target="#{{ item.rule|slugify }}_intended">
```

Comment:
I think this needs to be updated as well, otherwise when they go to copy, there will be an issue.